corretto/plan.sh

```diff
@@ -0,0 +1,61 @@
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_name=corretto
+# NOTE: Retrieve download link from here: https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html
+pkg_version=11.0.2.9.3
+pkg_source=https://d3pxv6yz143wms.cloudfront.net/${pkg_version}/amazon-corretto-${pkg_version}-linux-x64.tar.gz
```

Comment:
Just wondering if this cloudfront host is a reliable long-term host that we should commit? Seems like it might be a transient hostname that is valid for the moment.